packages/next-app/src/app/daoadmin/action/[actionId]/(components)/ActionAttributeContent.tsx

```diff
@@ -76,12 +122,18 @@ export function ActionAttributeContent() {
               <div className="flex flex-col gap-4">
                 {selectedAction?.fields.map((field) => {
                   return (
-                    <Input
-                      key={field.name}
-                      label={field.name}
-                      placeholder={field.placeholder}
-                      {...register(toSlug(`${field.name}`))}
-                    />
+                    <div key={field.name}>
+                      <Input
+                        label={field.name}
+                        placeholder={field.placeholder}
+                        {...register(toCammelCase(`${field.name}`))}
```

Comment:
Since you're using `react-input-form` you could already leverage their validations to validate that the input value is a valid Pool address.